index.html

```diff
@@ -298,6 +298,57 @@ <h3>Double-sided Ruby, background</h3>
           In this example, <span lang="ja">おだのぶなが</span> serves as [=Furigana=], while <span lang="ja">"1534〜82"</span> is presented as an [=interlinear note=].
         </p>
       </section>
+
+      <section>
+	<h3>Zaima, background</h3>
+	<p>The primary purpose of zaima annotation is to indicate
```

Comment:
Maybe add a link to https://w3c.github.io/elreq/zaima/WhatIsZaima.html ?

Reply:
I'll review this with @r12a . I was reluctant to do so in the past since the pages focused on an experimental approach where &lt;div&gt; sections are applied to layer ruby annotations to simulate multilevel rubies. My thought was to wait for an official standard for multilevel rubies, which are still forthcoming I believe.  However, the informative sections that present the Zaima background might be appropriate, the experimental markup could be removed from the use cases to avoid confusion.